firstvoices/backend/admin/base_admin.py

```diff
@@ -154,7 +154,7 @@ def get_site_from_object(self, request):
         if not hasattr(self, "parent_site"):
             self.parent_site = None
             object_id = request.resolver_match.kwargs.get("object_id")
-            instance = self.parent_model.objects.get(id=object_id)
+            instance = self.parent_model.objects.filter(id=object_id)
```

Comment:
Would this not return a `queryset/list` ? and probably both 'if' clauses below this wouldn't work as they are both testing the `instance` type but the type would always be a `queryset/list` ?

Reply:
Good call, thanks! I have updated this to return the first instance.